backend/README.md

```diff
@@ -6,7 +6,7 @@ All commands mentioned in this section are run from the `backend` directory unle
 
 ### Prerequisites
 
-- Java 21 installed on your system
+- [Java 21](https://www.oracle.com/java/technologies/downloads/) installed on your system
```

Comment:
I don't think that we should link to or recommend using the JDK from Oracle which is known for complicated licensing schemes. There are many distributions: for our Docker images, we use eclipse-temurin, but there are many more.

Reply:
I use Temurin, installed through [SDKMAN!](https://sdkman.io/) and can recommend it.